src/main/services/sealedClaimPdfGenerator.ts

```diff
@@ -0,0 +1,30 @@
+import * as express from 'express'
+
+import { Claim } from 'claims/models/claim'
+import { DocumentsClient } from 'documents/documentsClient'
+import { pdfEndpointResponseHandler } from 'services/pdfEndpointsResponseHandler'
+
+const documentsClient: DocumentsClient = new DocumentsClient()
+
+export class SealedClaimPdfGenerator {
```

Comment:
I just followed the "pattern" I have already found in the code. 

Shall I also inline claiman-copy service?

Reply:
Do not bother, I have done it here: https://github.com/hmcts/cmc-citizen-frontend/pull/561

Reply:
By the way there was no "pattern", we had to extract that common logic into helper handler because we were unable to use one route that downloads issue receipt in both claim issue and first contact journeys ;)